CONTRIBUTE.md

```diff
@@ -0,0 +1,56 @@
+
+# Run the demo environment 
+
+# Step 1: Setup and Compile
+
+* Compile the yard repository with **mvn clean install**
+    * https://github.com/kiegroup/yard
+* Compile this project **mvn clean install**
+* In order to run the Backstage plugin the repository with the Akrivis plugin is required
+  * Current Backstage plugins repository 
+    * https://github.com/kiegroup/backstage-plugins
+    * Branch name: **akrivis**
+  * It might be necessary to run **yarn install** in Backstage plugins repository
+
+# Step 2:
+
+## Option 1: Start all with tmux
+* Install **tmux** if you do not already have it
+* Run **./start.sh**
+* [Check Backstage Connection Workaround](#backstage-connection-workaround)
```

Comment:
there is missing also the follow up steps from Option 2
like there is required to have Akrivis buildable with java 21 while Yard is set to java 17 needed (i tried with 21 and it failed in tests due that ;) )

so i would add here at least to be ready with Maven and java 21 ;)
and then add here 
* The Backstage instance can now be opened from **http://localhost:3000/**

Reply:
That test is pretty annoying. I have been wondering if I should just make it runnable with any version. Since one version makes a numberic value an integer and another a double and then that does not match. This behaviour is caused by a dependency we use, so only solution to pass all the time would be to make the test a bit dirty looking.